cli/lookup.ts

```diff
@@ -269,6 +344,7 @@ export const command = new Command()
           ? "Successfully fetched all objects."
           : "Successfully fetched the object.",
       );
+      await writeOutput();
```

Comment:
This function should be called only if `output` option is specified. 
As you defined above, `console.error` could be called even though we actually don't want file output